### DISCUSSION TITLE
Allow more freedom for default highlights

### DISCUSSION THREAD

**Discussion by Felanbird** [Chatterino/chatterino2#3374] [Ideas]
In the development of Chatterino we've made certain decisions for users based on the consensus of developers, but some of these decisions removed user customization freedom. Specifically the ability to enable/disable: Show in mentions, Flash Taskbar, & Play Sounds for default highlights.

Should we allow for free customization of these default highlights? Since the user is already, or will become, familiar with the check-boxes based on custom highlights. (These new additions would all be disabled by default.)
![chatterino_default_highlights](https://user-images.githubusercontent.com/41973452/143724580-5bd33729-aba8-4da7-9892-d874ad17b816.png)

From personal experience I know that the code additions for Play Sound & Flash Taskbar are basically copy paste, but I'm unsure about Show in Mentions.

Relevant: https://github.com/Chatterino/chatterino2/issues/2824 - #3370

**Comment by leibnaurd** [comment]
I think that, as a mod, having the sound notification play for first time chatters would be very helpful.
